stream_alert_cli/test.py

```diff
@@ -118,18 +122,18 @@ def test_processor(self, rules):
             # Go over the records and test the applicable rule
             for test_record in test_records:
                 if not self.check_keys(rule_name, test_record):
-                    all_tests_passed = False
+                    self.passing = False
                     continue
 
                 self.apply_helpers(test_record)
 
                 # Run tests on the formatted record
-                alerts, expected_alerts = self.test_rule(
+                alerts, expected_alerts, all_valid_logs = self.test_rule(
```

Comment:
what's a "valid" log? is all_valid_logs intended to reflect if all received `records` matched a defined schema? please update the variable name if so